11/Zaubeerer/generators.py

```diff
@@ -0,0 +1,55 @@
+"""
+Turn the following unix pipeline into Python code using generators
+
+$ for i in ../*/*py; do grep ^import $i|sed 's/import //g' ; done | sort | uniq -c | sort -nr
+   4 unittest
+   4 sys
+   3 re
+   3 csv
+   2 tweepy
+   2 random
+   2 os
+   2 json
+   2 itertools
+   1 time
+   1 datetime
+"""
+
+import glob, os
+import re
+
+import collections
+
+def gen_files(pat):
+    """Finds all files"""
+
```

Comment:
remove blank lines inside efunctions

Reply:
pep does not complain, I would only use them in functions that are > 5 lines, not needed here